copy-plugin.js

```diff
@@ -10,8 +10,46 @@ const iosDir = './ios/plugins/'
 
 const platforms = [androidDir, iosDir]
 const pluginManifests = {
-  buysell: [],
-  spend: []
+  buysell: [
+    {
+      pluginId: 'co.edgesecure.libertyx',
```

Comment:
I would drop the `co.edgesecure` part and just go with the provider's website, like `com.libertyx`.

Reply:
done